libs/importer/src/components/lastpass/import-lastpass.component.ts

```diff
@@ -230,6 +230,28 @@ export class ImportLastPassComponent implements OnInit, OnDestroy {
     this.transformCSV();
   }
 
+  private getOidcRedirectUrlWithParams(oidcCode: string, oidcState: string) {
+    const redirectUri = this.oidcClient.settings.redirect_uri;
+    const params = "code=" + oidcCode + "&state=" + oidcState;
+    if (redirectUri.indexOf("bitwarden://") === 0) {
+      return redirectUri + "/?" + params;
+    } else {
+      return redirectUri + "&" + params;
+    }
```

Comment:
```suggestion
    if (redirectUri.indexOf("bitwarden://") === 0) {
      return redirectUri + "/?" + params;
    }
    
    return redirectUri + "&" + params;
```

This `if/else` statement could be simplified.